### PR TITLE
Fixed some layout issues

### PR DESCRIPTION
**Pull request by mihaelamj**
- Fixed main stackview layout
- All buttons are shown now

### CODE REVIEW

File: NGUIButtonInsetsExample/ViewController.swift [hunk 1]
```diff
@@ -274,44 +289,104 @@ final class ButtonView: UIView {
...
-        stackView.axis = .vertical
-        stackView.spacing = 50
+// MARK: Setup -
 
-        stackView.translatesAutoresizingMaskIntoConstraints = false
+private extension ViewController {
+    
+    func setupView() {
+        view.backgroundColor = .white
+        setupStackview()
```
Comment (noahsark769) [inline NGUIButtonInsetsExample/ViewController.swift]: Could we make the naming consistent? This would be `setupStackView` (with a capital V)

File: NGUIButtonInsetsExample/ViewController.swift [hunk 2]
```diff
@@ -274,44 +289,104 @@ final class ButtonView: UIView {
...
+        return stv
+    }()
+    
+    private lazy var allInsetsView: AllInsetsView = {
+        let aiv = AllInsetsView(insetsDidChange: { [weak self] type, insets in
+            guard let `self` = self else { return }
+            self.buttonsView.set(insets: insets, type: type)
+        })
+        aiv.translatesAutoresizingMaskIntoConstraints = false
+        aiv.accessibilityLabel = "AllInsetsView"
```
Comment (noahsark769) [inline NGUIButtonInsetsExample/ViewController.swift]: To be completely honest, I'm not as familiar with iOS accessibility as I'd like to be. However, as I understand it, the only views that need accessibility labels are the ones which will be read by VoiceOver. Is that true? In this case, I think these views which are implementation details of the UI (`allInsetsView`, `stackView`, `buttonsView`) don't need accessibility labels, and in fact, having accessibility labels might make the interface more confusing to use in VoiceOver.

It's possible you might be more familiar with accessibility than I am, so please let me know if I'm wrong, but could we remove these?

File: NGUIButtonInsetsExample/ViewController.swift [hunk 3]
```diff
@@ -274,44 +289,104 @@ final class ButtonView: UIView {
...
-            self.buttonView.set(insets: insets, type: type)
-        }))
-
+        view.safeAreaLayoutGuide.topAnchor.constraint(equalTo: stackView.topAnchor).isActive = true
+        view.safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: stackView.bottomAnchor).isActive = true
+    }
+    
+    func setupButtonsView() {
+        stackView.addArrangedSubview(buttonsView)
+//        buttonsView.heightAnchor.constraint(equalToConstant: 300).isActive = true
```
Comment (noahsark769) [inline NGUIButtonInsetsExample/ViewController.swift]: Let's remove this commented out line 👍

**Review by noahsark769 — CHANGES_REQUESTED** [review]
@mihaelamj sorry for the delay, I didn't see this! This looks great, thanks for taking the time to make these improvements. I'm happy to merge if we can make a few small changes. Thanks again!

File: NGUIButtonInsetsExample/ViewController.swift (hunk 1 above)
Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: Sure

File: NGUIButtonInsetsExample/ViewController.swift (hunk 2 above)
Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: I added accessibility labels solely for the purpose of debugging. I use Reveal app and it shows the views' names fro the accessibilityLabel property. 
Those `debug` names should not go into production for they don't help the user. I shall remove them.

File: NGUIButtonInsetsExample/ViewController.swift (hunk 3 above)
Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: sure

**Comment by mihaelamj** [comment]
Thanks for the feedback. I made the suggested changes.
Cheers.

File: NGUIButtonInsetsExample/ViewController.swift [hunk 4]
```diff
@@ -14,13 +14,13 @@ import UIKit
...
     private var timers: [Timer] = []
-
+    
     func addTimer(_ block: @escaping () -> Void) {
         timers.append(Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true, block: { _ in
             block()
         }))
     }
-
+
```
Comment (noahsark769) [inline NGUIButtonInsetsExample/ViewController.swift]: One last thing - looks like we added unnecessary whitespace here. Could we remove that? I believe there's a setting in Xcode preferences to trim trailing whitespace. Thanks!

**Review by noahsark769 — CHANGES_REQUESTED** [review]
Thanks again! @mihaelamj one last change and we'll merge this 👍

File: NGUIButtonInsetsExample/ViewController.swift (hunk 2 above)
Comment (noahsark769) [reply NGUIButtonInsetsExample/ViewController.swift]: Nice! For what it's worth, I think Reveal will show `accessibilityIdentifier`s as well, so that could be another option for the future (accessibility identifiers don't get read during VoiceOver)

File: NGUIButtonInsetsExample/ViewController.swift (hunk 4 above)
Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: Sure

File: NGUIButtonInsetsExample/ViewController.swift [hunk 5]
```diff
@@ -202,26 +215,26 @@ final class AllInsetsView: UIStackView {
         super.init(frame: .zero)
         self.axis = .vertical
         self.spacing = 30
-
+
```
Comment (noahsark769) [inline NGUIButtonInsetsExample/ViewController.swift]: Sorry, to be clear, there are a bunch of lines in this file which have trailing whitespace (including this one).  I think you should be able to check them out by looking at the github diff. We should remove the trailing whitespace from all of these 👍

**Review by noahsark769 — CHANGES_REQUESTED** [review]

File: NGUIButtonInsetsExample/ViewController.swift (hunk 5 above)
Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: Oh, I see, sure

Comment (mihaelamj) [reply NGUIButtonInsetsExample/ViewController.swift]: I have a new Mac, so I forgot that setting is not the default. I set it up now.

**Comment by noahsark769** [comment]
Looks great! Will merge, thanks again @mihaelamj for contributing!
